Data/Graph.hs

```diff
@@ -298,8 +356,35 @@ graphFromEdges' x = (a,b) where
 
 -- | Build a graph from a list of nodes uniquely identified by keys,
 -- with a list of keys of nodes this node should have edges to.
--- The out-list may contain keys that don't correspond to
--- nodes of the graph; they are ignored.
+--
+-- - @node@ is the type of data held at each vertex
+-- - @key@ is uniquely identifies each node in the graph
+--
+-- @(graph, nodeFromVertex, vertexFromKey) = graphFromEdges edgeList@
+--
+-- * @graph@ is the raw, array based adjacency list for the graph.
+-- * @nodeFromVertex@ returns the node associated with the given 0-based @Int@
+--   vertex, see /warning/ below.
```

Comment:
That comma should be a semicolon; the emphasis looks weird.

Reply:
Done.